src/xcode/ENA/ENA/Source/Services/Event Registration/__tests__/QRCodePlaygroundTests.swift

```diff
@@ -0,0 +1,25 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class QRCodePlaygroundTests: XCTestCase {
+
+    func testQRCodeGeneration() throws {
+		let qrCodeImage = try XCTUnwrap(QRCodePlayground.generateQRCode(with: "HTTPS://CORONAWARN.APP/E1/BIPEY33SMVWSA2LQON2W2IDEN5WG64RAONUXIIDBNVSXILBAMNXRBCM4UQARRKM6UQASAHRKCC7CTDWGQ4JCO7RVZSWVIMQK4UPA.GBCAEIA7TEORBTUA25QHBOCWT26BCA5PORBS2E4FFWMJ3UU3P6SXOL7SHUBCA7UEZBDDQ2R6VRJH7WBJKVF7GZYJA6YMRN27IPEP7NKGGJSWX3XQ"))
```

Comment:
CAPSLOCK IS CRUISE CONTROL FOR COOL!!1!

Reply:
Fun fact to have an Alphanumeric QR-Code, which will need less pixels, EVERYTHING HAS TO BE IN CAPSLOCK! :D